source/agora/utils/Utility.d

```diff
@@ -227,3 +227,47 @@ public struct UbyteHexString
         });
     }
 }
+
+/// Utility class to implement vibe.d ResultSerializer.serialize and
+/// ResultSerializer.serialize templates
+public class ResultSerializerUtil
+{
+    /***************************************************************************
+
+        Serialize the string as it is without any modifications
+        like JSON escaping
+
+        Params:
+            Policy: serialization policy type
+            InputT: input data type (string)
+            OutputRangeT: output range type
+            output_range: output range to serialize the string into
+            input_string: string to serialize
+
+    ***************************************************************************/
+
+    public static void raw_string_serialize(alias Policy, InputT, OutputRangeT)(OutputRangeT output_range, InputT input_string)
+    {
+        import std.string;
+        output_range ~= input_string.representation;
```

Comment:
Since it's a range, you should call `put` on it. IDK if there's a version that takes another range though.